com.unity.render-pipelines.universal/ShaderLibrary/Lighting.hlsl

```diff
@@ -605,6 +779,10 @@ half4 UniversalFragmentPBR(InputData inputData, half3 albedo, half metallic, hal
 #endif
 
     color += emission;
+
+#if defined(CC_DEBUG)
+    color = debugColor;
+#endif
```

Comment:
Is the debug intentional to commit?

Reply:
no I will remove it before the final commit and it's disabled by default.